<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenEXROptionsDialog</class>
 <widget class="QDialog" name="OpenEXROptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenEXR Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Floating Point Precision:</string>
       </property>
       <property name="buddy">
        <cstring>halfFloatRadioButton</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="halfFloatRadioButton">
       <property name="text">
        <string>16 bit</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="singleFloatRadioButton">
       <property name="text">
        <string>32 bit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="depthChannelCheckBox">
       <property name="text">
        <string>Include Z Buffer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Compression Type:</string>
       </property>
       <property name="buddy">
        <cstring>compressionTypeComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="compressionTypeComboBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Run length encoding</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PIZ Wavelet (recommended)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ZIP style (per scanline)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pixar 24bit ZIP (lossy for 32bit floats)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Uncompressed</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>halfFloatRadioButton</tabstop>
  <tabstop>singleFloatRadioButton</tabstop>
  <tabstop>depthChannelCheckBox</tabstop>
  <tabstop>compressionTypeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpenEXROptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpenEXROptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
